File: user-service/src/main/java/com/microservice/user/config/MongoConfiguration.java
package com.microservice.user.config;

import com.microservice.user.repository.UserRepository;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.mongodb.config.EnableMongoAuditing;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@Configuration
@EnableMongoAuditing
@EnableMongoRepositories(basePackageClasses = UserRepository.class)
public class MongoConfiguration {
}

File: kotlin-service/src/main/kotlin/com/microservice/kotlin/config/MongoConfiguration.kt
package com.microservice.kotlin.config

import com.microservice.kotlin.repository.TaskRepository
import org.springframework.context.annotation.Configuration
import org.springframework.data.mongodb.config.EnableMongoAuditing
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@Configuration
@EnableMongoAuditing
@EnableMongoRepositories(basePackageClasses = [TaskRepository::class])
class MongoConfiguration()
